Octokit/Models/Response/PullRequest.cs

```diff
@@ -48,6 +49,11 @@ public PullRequest(Uri url, Uri htmlUrl, Uri diffUrl, Uri patchUrl, Uri issueUrl
         }
 
         /// <summary>
+        /// The universal Id (different from the pull request number) this pull request.
```

Comment:
Can we please reword this to "The internal Id for this pull request (not the pull request number)." ?

And could you edit the comment on `Issue.Id` to be similar?  "The internal Id for this issue (not the issue number)."

Thanks 😀